### PR TITLE
Add logo in start page

### PR DESCRIPTION
**Pull request by xyang0089**
image

### CODE REVIEW

**Comment by dallasbrittany** [comment]
I wonder if maybe the logos would look better in the white part so we can see them more clearly? Also honestly I think it would look best without the Capital One logo. Having theirs centered would look really nice -- maybe just above the text but still in the same white box as the text? (Plus, I'm not sure we're allowed to actually put the C1 logo here like we discussed wanting to. We might be able to use it on the linkout to C1 resources though.)

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+16
```
Comment (dallasbrittany) [inline .nvmrc:1]: Thanks for adding this file! We might also want to be more specific if we're adding this. I'm using `16.20.2` specifically.

File: .env [hunk 2]
```diff
@@ -1,3 +1,4 @@
 PORT=3000
+NODE_OPTIONS=--openssl-legacy-provider
```
Comment (dallasbrittany) [inline .env:2]: Is this necessary?

File: package.json [hunk 3]
```diff
@@ -55,4 +55,4 @@
     "enzyme-to-json": "^3.6.1",
     "react-test-renderer": "^16.14.0"
   }
-}
+}
```
Comment (dallasbrittany) [inline package.json:58]: Did you mean to change anything in this file? We may need to just revert this file if not.

File: src/App.tsx [hunk 4]
```diff
@@ -53,7 +53,7 @@ const App = () => {
   const location = useLocation();
 
   const [state, setState] = useState<stateType>({
-    loggedin: false,
+    loggedin: true,
```
Comment (dallasbrittany) [inline src/App.tsx:56]: I think maybe you didn't mean to commit this and it's for debug?

File: src/assets/images/CapitalOneLogo.png
Comment (dallasbrittany) [inline src/assets/images/CapitalOneLogo.png:1]: We'll have to confirm we're allowed to commit this to this public repo. I don't know the rules.

File: src/components/shared-components/Modal.tsx [hunk 5]
```diff
@@ -134,4 +134,4 @@ const CustomModal = ({ header, body, actions, close }: any) => {
     </Wrapper>
   );
 };
-export { Modal, InfoModal, CustomModal };
+export { Modal, InfoModal, CustomModal };
```
Comment (dallasbrittany) [inline src/components/shared-components/Modal.tsx:137]: What actually changed here?

File: src/pages/Startpage.tsx [hunk 6]
```diff
@@ -1,5 +1,6 @@
+import { bannerlogo, capitalOneLogo } from 'assets';
 import React from 'react';
-import { Wrapper, Desc, DescText } from 'style/home';
+import { Wrapper, Desc, DescText, LogoContainer,Logo } from 'style/home';
```
Comment (dallasbrittany) [inline src/pages/Startpage.tsx:3]: Needs a space before Logo

File: src/pages/Startpage.tsx [hunk 7]
```diff
@@ -16,25 +17,23 @@ const Startpage = ({
...
-                    which high school students actively learn how to make better
-                    financial decisions and gain knowledge of budgeting, saving,
-                    and spending.
-                  </DescText>
-                </Desc>
-                {/* <Logoutbutton onClick={() => logout()}> sign out </Logoutbutton> */}
-              </>
-            </Wrapper>
-          </div>
+      {true ? (
```
Comment (dallasbrittany) [inline src/pages/Startpage.tsx:20]: Did you mean to have `true` or is that debug?

File: src/assets/images/CapitalOneLogo.png
Comment (dallasbrittany) [reply src/assets/images/CapitalOneLogo.png:1]: We should go ahead and delete this. We confirmed we can't use it in this app.

File: src/pages/Startpage.tsx [hunk 8]
```diff
@@ -16,25 +17,23 @@ const Startpage = ({
...
+          <Desc>
+            <div className="desc-title">Dollars & $ense Reality Fair</div>
+            <DescText>
+              The United Way of Northeast Louisiana Dollars & $ense Reality Fair is a financial
+              education simulation during which high school students actively learn how to make
+              better financial decisions and gain knowledge of budgeting, saving, and spending.
+            </DescText>
+          </Desc>
+          {/* Uncomment this to enable logout */}
+          {/* <Logoutbutton onClick={() => logout()}> sign out </Logoutbutton> */}
```
[the remark this answers is not in the document]
Comment (xyang0089) [reply src/pages/Startpage.tsx:35]: Yeah sorry. will revert this

File: src/pages/Startpage.tsx (hunk 7 above)
Comment (xyang0089) [reply src/pages/Startpage.tsx:20]: I thought I didnt commit this. My bad

**Comment by xyang0089** [comment]
opening a new pr this is too much clean up

**Pull request closed without merging**